test/algorithms/eigen_solvers/test_qeom_ee.py

```diff
@@ -18,8 +18,8 @@
 from test import QiskitNatureTestCase
 import numpy as np
 
```

Comment:
Corresponding to my earlier comment, the module `test.algorithms.eigen_solvers` can be removed :+1: